### PR TITLE
Fix highlight layer properties in inspector

### PR DESCRIPTION
**Pull request by sebavan**
https://forum.babylonjs.com/t/highlightlayer-showing-invalid-parameters-in-the-inspector/45863/2

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/14541/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14541/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14541/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/14541/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/14541/merge
https://gui.babylonjs.com/?snapshot=refs/pull/14541/merge
https://nme.babylonjs.com/?snapshot=refs/pull/14541/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/14541/merge#BCU1XR#0

**Review by RaananW — APPROVED** [review]

**Review by Popov72 — APPROVED** [review]

File: packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/layers/layerPropertyGridComponent.tsx [hunk 1]
```diff
@@ -39,28 +42,48 @@ export class LayerPropertyGridComponent extends React.Component<ILayerPropertyGr
...
+                    {layer.getClassName() === HighlightLayer.EffectName && (
+                        <div className="fragment">
+                            <CheckBoxLineComponent
+                                label="Inner Glow"
+                                target={layer}
+                                propertyName="innerGlow"
+                                onPropertyChangedObservable={this.props.onPropertyChangedObservable}
+                            />
+                            <CheckBoxLineComponent
+                                label="Outter Glow"
```
Comment (carolhmj) [inline packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/layers/layerPropertyGridComponent.tsx:80]:
```suggestion
                                label="Outer Glow"
```

**Review by carolhmj — COMMENTED** [review]
Just a word nit:

**Pull request merged**